lib/get-files.js

```diff
@@ -135,13 +145,22 @@ export async function docker(path, {
   ).createFilter()
 
   const prefixLength = path.length + 1
-  const accepts = function (file) {
+  const accepts = async function (file) {
```

Comment:
Seems like this `accepts` function should be DRY'd up a bit, but that's unrelated to this PR.